frontend/src/store/modules/dataSource.ts

```diff
@@ -109,15 +108,6 @@ const actions = {
       dataSource: createdDataSource,
     });
 
-    if (createdDataSource.databaseId !== UNKNOWN_ID) {
-      // Refresh the corresponding database as it contains data source.
-      dispatch(
-        "database/fetchDatabaseById",
```

Comment:
I take a look into database list/detail page, they always fetch the latest data when mounting. So we do not need to fetch database by id here which could cause `not found`  error with wildcard database id.

Reply:
@d-bytebase here. After patched data source, call `fetchDatabaseById` with a wildcard database id. 

So the question is how should we handle getting database by a wildcard(*) id, not found or database?